src/DefaultBuilder/test/Microsoft.AspNetCore.FunctionalTests/WebHostFunctionalTests.cs

```diff
@@ -151,7 +151,7 @@ public void LoggingConfigurationSectionPassedToLoggerByDefault()
             }
         }
 
-        [ConditionalFact]
+        [ConditionalFact(Skip = "https://github.com/aspnet/AspNetCore-Internal/issues/2170")]
         [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]
```

Comment:
Minor but this is really `[Fact(Skip = "https://github.com/aspnet/AspNetCore-Internal/issues/2170")]` and there's no need for the `[OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]`.